server/models/Request.js

```diff
@@ -31,10 +31,15 @@ const requestSchema = new Schema({
     enum: ["waiting", "matched", "referred"],
     required: true,
   },
-  scale: {
-    type: Number,
+  created_date: {
+    type: Date,
     required: true,
-  }
+    default: Date.now,
+  },
+});
+
+requestSchema.virtual("scale").get(function() {
+  return -this.created_date.getTime() - 7*24*60*60*this.priority;
```

Comment:
hình như cái chỗ này là nó sẽ automatically tạo ra một field mới với cái function này đúng ko. Nhưng mà function này hình như chưa complete hả. It should be maching day -created_date - priority, đúng ko